src/env.mjs

```diff
@@ -32,6 +32,20 @@ export const env = createEnv({
    */
   client: {
     // NEXT_PUBLIC_CLIENTVAR: z.string().min(1),
+    NEXT_PUBLIC_SITE_URL: z
+      .optional(z.string())
+      .transform((str) => {
+        let url =
+          str ??
+          process.env.NEXT_PUBLIC_VERCEL_URL ?? // Automatically set by Vercel.
+          "http://localhost:3000/";
+        // Make sure to include `https://` when not localhost.
+        url = url.includes("http") ? url : `https://${url}`;
+        // Make sure to include a trailing `/`.
+        url = url.charAt(url.length - 1) === "/" ? url : `${url}/`;
```

Comment:
Suggestion: Similar to the comment above, we can consider url.endsWith("/") in this case since we are always checking the end of the string

Reply:
realised this is not necessary, removed at [`aa17b9a#L45`](https://github.com/AfterClass-io/afterclass.io-v2/pull/132/commits/aa17b9a219d9c9b5bef08309a784cdcdc479a0a3#diff-4ed47d4643b7ec88e80c88221860b4c9eca9be1cb89d980ab40ed6087ae47caaL45)

Reply:
Ok, thanks for the update!